alertlogic-cli

```diff
@@ -9,12 +9,15 @@ import traceback
 
 sys.path.insert(0, os.path.dirname(__file__))
 
+
 def import_dependencies():
     try:
         global requests
         import requests
     except ImportError:
-        print("requests library required, install it first: pip install requests")
+        print(
+            "requests library required, install it first: pip install requests"
```

Comment:
It is kind of questionable formatting. But besides that, it is inconsistent with L27 and L36.